mlflow/tracking/fluent.py

```diff
@@ -387,7 +390,12 @@ def search_runs(experiment_ids=None, filter_string="", run_view_type=ViewType.AC
         data['params.' + key] = params[key]
     for key in tags:
         data['tags.' + key] = tags[key]
-    return pd.DataFrame(data)
+
+    df = pd.DataFrame(data)
+    df['start_time'] = pd.to_datetime(df['start_time'], unit="ms", utc=True)
```

Comment:
Can we move `pd.to_datetime(...)` up to where `start_time` is appended to the info dictionary? Or are there large performance benefits to doing it after the dataframe is created?

Something along the lines of 
`info['start_time'].append(pd.to_datetime(run.info.start_time, units="ms", utc=True))`

Reply:
Done